jsemanticdb/src/main/protobuf/bazel_query.proto

```
@@ -0,0 +1,57 @@
+// the numbers come from Bazel.
+//
+// Consumed by scala.meta.internal.metals.mbt.importer.BazelStreamedProto (in the
+// `metals` module, which gets the generated classes transitively). It lives here
+// rather than in `metals` so protoc generation stays in the Java-gen modules and
+// off `metals`.
+//
+// Based on
+// https://github.com/bazelbuild/bazel/blob/master/src/main/protobuf/build.proto
+syntax = "proto2";
+
+package metals_bazel_query;
```

Buf (1.71.0)

[error] Files with package "metals_bazel_query" must be within a directory "metals_bazel_query" relative to root but were in directory "jsemanticdb/src/main/protobuf".

(PACKAGE_DIRECTORY_MATCH)